src/org/traccar/protocol/Tk103ProtocolEncoder.java

```diff
@@ -16,36 +16,89 @@
  */
 package org.traccar.protocol;
 
+import org.traccar.Context;
 import org.traccar.StringProtocolEncoder;
 import org.traccar.helper.Log;
 import org.traccar.model.Command;
 
 public class Tk103ProtocolEncoder extends StringProtocolEncoder {
 
+    protected String t580WFormatCommand(Command command, String format, String... keys) {
```

Comment:
Why do we need this method? I suggest we remove it.

Reply:
done